aim/sdk/__init__.py

```diff
@@ -1,18 +1,4 @@
 # Main functions
 from aim.sdk.track import track, set_params
 from aim.sdk.flush import flush
-from aim.sdk.init import init
-from aim.sdk.select import select_metrics, select_runs
```

Comment:
forward to runs and metrics new search interfaces? +add deprecation warnings

Reply:
> what do you think about moving `aim/storage/sdk` to `aim/sdk`?

totally makes sense. but only after current `aim/sdk` contents are finally dropped.